src/mono/browser/runtime/invoke-js.ts

```diff
@@ -2,12 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 /* eslint-disable prefer-rest-params */
 
-<<<<<<< HEAD
 import NativeAOT from "consts:nativeAOT";
```

Comment:
```suggestion
import NativeAOT from "consts:nativeAOT";

```
Future conflict prevention.